deployments/helm/KubeArmorOperator/crds/operator.kubearmor.com_kubearmorconfigs.yaml

```diff
@@ -57,6 +57,12 @@ spec:
                 type: string
               defaultVisibility:
                 type: string
+              enableStdOutAlerts:
+                type: string
```

Comment:
Is it possible to use the type `boolean` instead? Though I think we'll need to convert them to string when specifying as an env var but user might face inconvenience if they specify these without the quotes. WDYT? :thinking:

Reply:
I guess we can convert the type to boolean, but we can specify in the documentation to use the parameters as string, as we mostly specify the values in the form of string.

Reply:
Ig we should use boolean, because from an idiomatic point of view string could open the possibility to invalid values but not in the case of bool

Reply:
I guess we need to regenerate the operator config to convert it to bool right?

Reply:
@VedRatan Yeah, right. It might be a little tedious to regenerate and make the changes but the effort will be worth it : )

Reply:
Commit incoming.